README.md

```diff
@@ -24,3 +24,21 @@ translators in seconds.
 Zanata is Free software, licensed under the [LGPL][].
 
 [LGPL]: http://www.gnu.org/licenses/lgpl-2.1.html
+
+Developers
+----------
+
+You can use one of the scripts to build the project using maven:
+
+[`etc/scripts/quickbuild.sh`](etc/scripts/quickbuild.sh) - Builds the project as quickly as possible, targeting all
+browsers in the GWT components, and skipping all checks and verifications (i.e.
+tests, checkstyle, etc)
+
+If you wish to build GWT componets for chrome or firefox, you can specify the
+`-c` and `-f` arguments respectively.
+
+The `-f` argument prints the script's help.
+
+[`etc/scripts/cargowait.sh`](etc/scripts/cargowait.sh) - Builds the Zanata artifact and starts a JBoss server using the
+cargo plugin. This script is particularly useful for quickly starting a Zanata
+instance, and for running functional tests from an IDE.
```

Comment:
Well, there's not much point in encouraging people to run a Zanata instance this way if they won't be able to log in. Perhaps we should just say it's useful for running individual functional tests from an IDE.